schema/endpoint.proto

```diff
@@ -16,6 +16,7 @@ enum TxStatus {
   COMMITTED = 4;
   ON_PROCESS = 5;
   NOT_RECEIVED = 6;
+  EXPIRED = 7;
```

Comment:
Is there an available conversion from proto to model and vice versa?

Reply:
Check 612cbe6, and I didn't find reverse way convertion